tsconfig.json

```diff
@@ -1,22 +1,27 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
-    "rootDir": "./",
     "outDir": "build",
-    "target": "esnext",
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
```

Comment:
Why do we have `allowJs` enabled?

Reply:
The output of the protos are .js files along with typing files so in order to consume them we need to enable allowJs